nodejs/week1/warm_up/warmup.js

```diff
@@ -0,0 +1,23 @@
+class Circle {
+    constructor(radius) {
+        this.radius = radius;
+    }
+
+    getDiameter() {
+        return 2 * this.radius;
+    }
+
+    getCircumference() {
+        return 2 * Math.PI * this.radius;
+    } 
+
+    getArea() {
+        return Math.PI * Math.pow(this.radius, 2);
+    }
```

Comment:
This looks great 👍

You may want to sort the methods alphabetically. That can make them easier to find. This is not required but you may encounter that different companies have styleguides which such recommendations.